src/modules/Dashboard/features/Participants/Participants.tsx

```diff
@@ -276,9 +284,12 @@ export const Participants = () => {
           <Checkbox
             checked={false}
             icon={
-              <StyledCheckBox>
-                <Svg id="checkbox-empty-outlined" height="20" width="20" />
-              </StyledCheckBox>
+              <Svg
+                fill={variables.palette.outline_variant2}
+                height="20"
+                id="checkbox-empty-outlined"
+                width="20"
+              />
```

Comment:
It seems we'll be standardizing on this styling of checkboxes at least in tables, and probably universally in the admin panel as a whole. Would it be worth creating a shared `Checkbox` component that defaults to this SVG for the `icon` prop?